ci-conda.Dockerfile

```
@@ -233,8 +233,18 @@ wget https://github.com/cli/cli/releases/download/v${GH_CLI_VER}/gh_${GH_CLI_VER
 tar -xf gh_*.tar.gz
 mv gh_*/bin/gh /usr/local/bin
 rm -rf gh_*
+
+# Install OpenTelemetry instrumentation
+pip install opentelemetry-distro[otlp] opentelemetry-exporter-prometheus
+curl -L -o "otel-cli-${CPU_ARCH}.tar.gz" https://github.com/equinix-labs/otel-cli/releases/download/v0.4.5/otel-cli_0.4.5_linux_${CPU_ARCH}.tar.gz
+tar -zxf  "otel-cli-${CPU_ARCH}.tar.gz"
+mv otel-cli /usr/local/bin/
+git clone -b add-conda-build-instrumentation https://github.com/msarahan/opentelemetry-python-contrib
```

```suggestion
git clone -b add-conda-build-instrumentation https://github.com/rapidsai/opentelemetry-python-contrib
```
we should probably fork this internally to RAPIDS somehow instead of using my personal fork.

---

to ensure there is no extra bloat in our Dockerfiles, can you be sure to remove the downloaded tarball and any other unnecessary files from this code block (and the others)?